app/models/concerns/user_access_key_overrides.rb

```diff
@@ -31,13 +30,11 @@ def valid_password?(password)
   end
 
   def unlock_user_access_key(password)
-    self.user_access_key = UserAccessKey.new(
+    self.user_access_key = Encryption::UserAccessKey.new(
       password: password,
       salt: authenticatable_salt,
       cost: password_cost
-    )
-    encrypted_key_maker.unlock(user_access_key, encryption_key)
-    user_access_key
+    ).unlock(encryption_key)
   end
 
```

Comment:
should `encrypted_key_maker` be removed? and should the comments in `sessions_controller_spec` that reference `EncryptedKeyMaker` be removed as well?

Reply:
Yes and yes, thanks